libraries/core_libs/network/src/tarcap/packets_handlers/common/ext_votes_packet_handler.cpp

```diff
@@ -53,6 +59,12 @@ void ExtVotesPacketHandler::sendPbftNextVotes(dev::p2p::NodeID const &peer_id,
       }
     }
   }
+
+  if (send_next_votes_bundle.size() - index > kMaxVotesInPacket) {
+    periodic_events_tp_->post(100, [this, index, peer_id, send_next_votes_bundle] {
+      sendPbftNextVotes(peer_id, send_next_votes_bundle, index + kMaxVotesInPacket);
```

Comment:
we should change it to move semantics for send_next_votes_bundle as we are making copy here

Reply:
no longer applies